sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -96,6 +96,7 @@ export class LogsQueryClient {
 // @public
 export interface LogsQueryClientOptions extends PipelineOptions {
     endpoint?: string;
+    scopes?: string | string[];
```

Comment:
Just wondering if we need this to be `string | string[]` today? Since it's an input type we can always expand it later unless we know a valid use case for supporting both today?

Reply:
This is just what TokenCredential supports (ie, I made no editorial decision here):

```typescript
interface TokenCredential {
  getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
}
```

[link](https://github.com/Azure/azure-sdk-for-js/blob/c4f51bee047ee069fb1a005d9132425372cec0c2/sdk/core/core-auth/src/tokenCredential.ts#L21)